web/frontend/app/entry.server.tsx

```diff
@@ -11,6 +11,9 @@ import { createReadableStreamFromReadable } from "@remix-run/node";
 import { RemixServer } from "@remix-run/react";
 import { isbot } from "isbot";
 import { renderToPipeableStream } from "react-dom/server";
+import { setBaseUrl } from "./remix-client/remix-client.mjs";
+
+setBaseUrl("http://composer.plt.local");
```

Comment:
@iamfortune how did you generate the client?

Are you calling the APIs from the frontend or only from the backend? It would be slightly more performant to avoid passing through the composer here, but it's needed atm.